src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -293,7 +293,7 @@ def notify_issue_update(testcase, status):
 
   if status in ('verified', 'wontfix'):
     logs.log(f'Closing issue {testcase.github_issue_num} '
-             f'in GitHub repo {testcase.github_id}: '
+             f'in GitHub repo {testcase.github_repo_id}: '
```

Comment:
Very odd that this wasn't caught by linting.

Reply:
Oliver and I decided to add some specific mock tests that will execute these functions later : )